packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -22,11 +22,22 @@ import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 import BpkText from 'bpk-component-text';
 import BpkButton from 'bpk-component-button';
-import BpkCard from 'bpk-component-card';
 
-import STYLES from './BpkGraphicPromo.module.scss';
+import CARD_STYLES from '../../bpk-component-card/src/BpkCard.module.scss';
 
-const getClassName = cssModules(STYLES);
+import GP_STYLES from './BpkGraphicPromo.module.scss';
+
+const getClassName = cssModules(GP_STYLES);
+const getCardClassName = cssModules(CARD_STYLES);
+
+const ACCESSIBILITY_KEYS = {
+  Enter: 13 /* Enter */,
+  Space: 32 /* Space */,
```

Comment:
Have you tried this with VoiceOver on Safari? As I know that in order to select a element its a combination of keys and not just space, so not sure if just space gets detected or the event that gets triggered specifically

Reply:
Thanks for bringing this up. Testing on Safari now. 🙂